src/foraging_gui/Foraging.py

```diff
@@ -3918,27 +3941,11 @@ def _Start(self):
                     self.Metadata_dialog._show_project_info()
                     logging.info('Setting Project name: {}'.format(project_name))
                     add_default = False
-            if add_default:
-                project_name = 'Behavior Platform'
-                logging.info('Setting Project name: {}'.format('Behavior Platform'))
-                projects = [self.Metadata_dialog.ProjectName.itemText(i)
-                            for i in range(self.Metadata_dialog.ProjectName.count())]
-                index = np.where(np.array(projects) == 'Behavior Platform')[0]
-                if len(index) > 0:
-                    index = index[0]
-                    self.Metadata_dialog.ProjectName.setCurrentIndex(index)
-                    self.Metadata_dialog._show_project_info()
-                else:
-                    project_info = {
-                        'Funding Institution': ['Allen Institute'],
-                        'Grant Number': ['nan'],
-                        'Investigators': ['Jeremiah Cohen'],
-                        'Fundee': ['nan'],
-                    }
-                    self.Metadata_dialog.project_info = project_info
-                    self.Metadata_dialog.ProjectName.addItems([project_name])
-            self.project_name = project_name
 
+            if self.add_default_project_name and add_default:
+                self._set_default_project()
```

Comment:
It should be `project_name = self._set_default_project()`, right?
And then that function should return `project_name`?

Reply:
that's right! Updated